packages/transport-queue/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@stellarjs/messaging-queue",
+  "name": "@stellarjs/transport-queue",
```

Comment:
Maybe it should include `base` in it's name or something similar? to hint that it cannot be used directly